### PR TITLE
Fixes the readme spelling

### PR DESCRIPTION
**Pull request by ferdox2**
- **What does this PR do?**
Fixes minimum errors in the readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,4 +72,4 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!
```
Comment (prateek3255) [inline README.md]: Except this, I don't think other fixes are required.

**Comment by ferdox2** [comment]
Done

**Comment by prateek3255** [comment]
@ferdox2 I think you misunderstood, I said except the change I mentioned all the other changes were not required. So you can add the one I pointed out and remove the rest

**Comment by ferdox2** [comment]
Sorry for the misunderstanding, I think it's done.

**Pull request merged**